distributions/otelcol-contrib/manifest.yaml

```diff
@@ -264,5 +264,3 @@ providers:
 
 # When adding a replace, add a comment before it to document why it's needed and when it can be removed
 replaces:
-  # see https://github.com/openshift/api/pull/1515
-  - github.com/openshift/api => github.com/openshift/api v0.0.0-20230726162818-81f778f3b3ec
```

Comment:
This is being replaced in other distributions too (ebpf-profiler). Could we remove it there too?